### ISSUE TITLE
g0blin stopped working after working once

### ISSUE THREAD

**Issue by nik312123** [Sticktron/g0blin#133]
Context: iPhone 6s with 32 GB and iOS 10.3.3

It took two tries, but I originally got g0blin working! Hurrah!

But then I rebooted... I haven't been able to get it to work since. One of the following always happen:

1. The g0blin app shows "exploit failed"
2. The iPhone crashes when I tap jailbreak.
3. (The rare time that it seems to work) The iPhone crashes when I tap respring

I have no idea where to go! I tried waiting 60 – 90 seconds, but it still doesn't work! Let me know if any other data is needed.

**Comment by D-an-W** [comment]
You need to use a different respring method then after #3 you should be good to go.

**Comment by nik312123** [comment]
I don't see anything in #3. Did you mean to refer to another issue?

**Comment by D-an-W** [comment]
I meant your number three in the first post.

**Comment by nik312123** [comment]
Ah, I see. Well, 3 rarely happens, but I'll keep trying. I'll put an update when it happens. However, even if this does work, is there a reason why it takes so many tries? That's an issue itself. (Unless this is typical. I'm a first time jailbreaker.)

**Comment by DirkT86** [comment]
Not really unfortunate man!  It's still in beta and I've seen a lot less reliable jailbreaks with a lot more work required in all the years I've been doing this!  I'm sure it'll get better in future releases

**Comment by burhangee** [comment]
The iPhone 5s 10.3.3 Is smooth, first time Jailbreak works ( after reboot too) but on iSE on 10.3.0 uffffff its pain in the ass trying to Jailbreak it, it says try again or reboots. When it does work, dont respring using g0blin. use Cydia. reinstall a tweak n Respring. but sometimes it takes 10 tries. better is try in airplane mode, turn on wifi... It seems to work for me in alot less tries.

**Comment by DirkT86** [comment]
It really seems every iOS version on every different phone is having different experiences with this jailbreak!  I've got the SE on 10.3.2 and I've only had it reboot maybe two or three times when I first started playing around with it over a week ago and since I started using Cydia to do the first respring I haven't had any problems

**Comment by nik312123** [comment]
For me, this works best one of the few times after I sideload g0blin via Cydia Impactor.

Then, I either try it immediately after it's sideloaded, or I reboot.

When I reboot, I then:

1. Wait 60-90 seconds
2. Open the g0blin app
3. Wait 3 more seconds
4. Tap jailbreak.

If the above doesn't work, I retry five times. If it doesn't work after those 5 times, I sideload g0blin again and restart the process.

If any of these times 'work,' meaning it shows **'respring,'** then I do NOT press respring like you mentioned, @DirkT86, and go into Cydia, install or reinstall a tweak, and press respring from Cydia.

[Here](https://yalujailbreak.net/errors-and-fixes-g0blin/#Q_The_app_keeps_saying_8220exploit_failed8221_every_time_I_run_it_Is_there_a_fix_for_this) is a guide for anyone who needs any tips for the jailbreak who looks at this issue. I'll close this issue since it is open in a few other locations.
